src/components/File.jsx

```diff
@@ -68,21 +85,28 @@ class File extends Component {
     const classes = classNames(styles['fil-content-file'], getClassFromMime(attributes))
     if (editing) {
       return (
-        <td class={classes}>
+        <td className={classes}>
           <FilenameInput name={attributes.name} onSubmit={val => this.edit(val)} />
         </td>
       )
     }
-    return isDir(attributes)
-      ? (
-        <td class={classes}>
+    if (isDir(attributes)) {
+      return (
+        <td className={classes}>
           <a onClick={() => onOpen(attributes.id)}>
             {attributes.name}
-            {attributes.isOpening === true && <div class={styles['fil-loading']} />}
+            {attributes.isOpening === true && <div className={styles['fil-loading']} />}
           </a>
         </td>
       )
-      : <td class={classes}><a target='_blank' href={`http://cozy.local:8080/files/download/${attributes.id}`}>{filename}<span class={styles['fil-content-ext']}>{extension}</span></a></td>
+    }
+    return (
+      <td className={classes}>
+        <a target='_blank' href={`http://cozy.local:8080/files/download/${attributes.id}`}>
```

Comment:
To the best of my knowledge, there is still work to do so that cozy-stack exposes that URL to the client. Additionally, this "download" feature is a temporary one, we have time to correct this.